app/database/schema.prisma

```diff
@@ -573,3 +573,14 @@ model Booking {
   from      DateTime? @db.Timestamptz(3)
   to        DateTime? @db.Timestamptz(3)
 }
+
+model Session {
+  id String @id @default(cuid())
+
+  expires DateTime?
+  userId  String?
+  data    Json
```

Comment:
Suggestion: normalise instead of hiding in a JSONB col?

Given the schema below:
```ts
 email: z.string(),
 userId: z.string(),
 expiresAt: z.number(),
 expiresIn: z.number(),
 accessToken: z.string(),
 refreshToken: z.string(),
```

we could have a col for:
`accessToken`, `refreshToken`

---
Later in the `createDatabaseSessionStorage` implementation, we could use the `expiresAt` from the supabase auth token to populate the `expires` col of this table?
Then, you control this settings from supabase auth dashboard.